programs/survival/destructive_lightning.sc

```diff
@@ -0,0 +1,44 @@
+
+//script that makes lightning slightly more destructive
+
+__config() -> m( l('scope', 'global'));
+
+
+__command() -> (
+    print(format('b \nDestructive Lighting'));
+    print('--------------------------------');
+    print('To enable destructive lightning:');
+    print(format(' /destructive_lightning start\n', '^ Click to start', '!/destructive_lightning start'));
+    print('To disable destructive lightning:');
+    print(format(' /destructive_lightning stop\n', '^ Click to stop', '!/destructive_lightning stop'))
+);
+
+
+__on_tick() -> (
+    if(global_started == 1,
+    e = entity_list('lightning_bolt'); //finds all lightning bolts
+        for(e,
+            look = l(0.0, -1.0, 0.0);
+            fireball_pos = (_ ~ 'pos')+look;  //finds lightning bolt position
+            if (global_last != query(_, 'uuid'),  //only summons one fireball per bolt
+                spawn('tnt', (_ ~ 'pos'));
+                game_tick(50);

```

Comment:
why do you need to run a game tick here?

Reply:
Forgot to remove it when I copied the code out of the loop and into the __on_tick event